docs/cep/table/index.md

```diff
@@ -5,6 +5,10 @@ title: Tables (Collections)
 
 A _table_ is a stored version of an stream or a table of events. Its schema is defined in the _table definition_ that is similar to a stream definition. These events are stored in database. In Macrometa GDN, tables are called [collections](../../collections/).
 
+:::note
+If you want to use an existing Macrometa collection as a store, then you still need to define it in the stream worker and you must enable streaming on the collection.
```

Comment:
@diana-macrometa - The store may not have associated stream. This is not mandatory in this case.

Reply:
Yes, this is true for a source with type='database'  but not for a table/store.